src/api.rs

```diff
@@ -463,13 +463,16 @@ impl Context {
 
   fn get_frame_properties(&mut self, idx: u64) -> Result<FrameInvariants, FrameInvariants> {
     if idx == 0 {
+      let mut seq = Sequence::new(&self.config.video_info);
```

Comment:
Would slightly prefer this as a parameter to new(), but can be changed separately.